### PR TITLE
Follow marshmallow's update

### PR DESCRIPTION
**Pull request by yukihiko-shinoda**
Hello, thank you for create and maintain amazing package.
would you review my pull request?

- New parameter "many" was added. 35f6f4c286241e7dec1fcbeb9912b02b3b4887ba
https://github.com/marshmallow-code/marshmallow/commit/35f6f4c286241e7dec1fcbeb9912b02b3b4887ba
https://github.com/marshmallow-code/marshmallow/pull/1220

- Dump validation was removed. a3e95aeaaab9ec649cda80ede73793f471531cc0
https://github.com/marshmallow-code/marshmallow/commit/a3e95aeaaab9ec649cda80ede73793f471531cc0
https://github.com/marshmallow-code/marshmallow/pull/1304

### CODE REVIEW

**Comment by Lordnibbler** [comment]
Can this be approved ASAP please? :)

**Comment by wintonzheng** [comment]
+1

**Comment by lidatong** [comment]
Hey there, thank you for putting in this fix!

File: dataclasses_json/mm.py [hunk 1]
```diff
@@ -282,7 +282,7 @@ def build_schema(cls: typing.Type[A],
                                  if field.name != 'dataclass_json_config')})
 
     @post_load
-    def make_instance(self, kvs):
+    def make_instance(self, kvs, **kwargs):
```
Comment (lidatong) [inline dataclasses_json/mm.py]: I assume `**kwargs` is intentionally ignored as part of the fix?

Comment (Lordnibbler) [reply dataclasses_json/mm.py]: Yes seems to be the same fix described here: https://github.com/marshmallow-code/flask-marshmallow/issues/139

**Comment by Lordnibbler** [comment]
@lidatong any chance we can get a new release for this fix?

**Comment by lidatong** [comment]
yep, doing that right now

**Comment by lidatong** [comment]
[Released](https://github.com/lidatong/dataclasses-json/releases/tag/v0.3.2)

**Comment by Lordnibbler** [comment]
Appreciate that! Thanks

**Pull request merged**